MvvmCross/Platform/Platform/Converters/MvxDictionaryValueConverter.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace MvvmCross.Platform.Converters
+{
+    public class MvxDictionaryValueConverter<TKey, TValue> : MvxValueConverter<TKey, TValue>
```

Comment:
Is it possible to have a a non-constrained version of the Converter?

Reply:
Could you please clarify this request.
At the moment, the `MvxDictionaryValueConverter` does not apply any constraints.